### PR TITLE
update One to ~2.0

### PR DESCRIPTION
**Pull request by lizhichao**
update One to ~2.0

### CODE REVIEW

File: php/one-fpm/App/index.php [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 define('_APP_PATH_VIEW_', __DIR__ . '/View');
 
+function uuid()
```
Comment (waghanza) [inline php/one-fpm/App/index.php]: what is that function for ?

File: php/one/App/swoole.php [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 define('_APP_PATH_VIEW_', __DIR__ . '/View');
 
+function uuid()
```
Comment (waghanza) [inline php/one/App/swoole.php]: what is this function for ?

File: php/one-fpm/App/index.php (hunk 1 above)
Comment (lizhichao) [reply php/one-fpm/App/index.php]: By default, the framework sets a unique id for logging. There is no need for logging here. I rewritten it.

File: php/one/App/swoole.php (hunk 2 above)
Comment (lizhichao) [reply php/one/App/swoole.php]: By default, the framework sets a unique id for logging. There is no need for logging here. I rewritten it.

**Comment by waghanza** [comment]
@lizhichao could you update files

+ `php/one-fpm/config.yaml`
+ `php/one/config.yaml`

to reflect  the updated `one` version ?

**Pull request merged**